tests/authentication.test.js

```diff
@@ -5,13 +5,97 @@ import customMessages from '../src/utils/customMessages';
 import statusCodes from '../src/utils/statusCodes';
 import mockData from './data/mockData';
 
-const { signupData, incompleteData } = mockData;
+const { 
+signupData, 
+invalidData, 
+invalidFirstname, 
+invalidLastname, 
+invalidEmail, 
+invalidPassword, 
+invalidAddress 
+} = mockData;
 
 chai.use(chaiHttp);
 chai.should();
 
-// Signup
+
 describe('User sign up', () => {
+  it('Should return 400 if certain body is invalid', (done) => {
+    chai
+      .request(server)
+      .post('/api/auth/signup')
+      .send(invalidData)
+      .end((err, res) => {
+        const { error } = res.body;
+        expect(res.status).to.equal(statusCodes.badRequest);
+        done();
+      });
+  });
+  it('Should return 400 if firstname is invalid', (done) => {
+    chai
+      .request(server)
+      .post('/api/auth/signup')
+      .send(invalidFirstname)
+      .end((err, res) => {
+        const { error } = res.body;
+        expect(res.status).to.equal(statusCodes.badRequest);
+        expect(error);
+        expect(error).to.equal('firstName must consists of alphabets only and no whitespace');
```

Comment:
Could you create these custom messages in customMessages.js and import them here? The same way you used statusCodes.